playbooks/adhoc/uninstall.yml

```diff
@@ -85,7 +85,7 @@
 
   - name: Remove packages
     package: name={{ item }} state=absent
-    when: not is_atomic | bool
+    when: not is_atomic | bool and openshift_remove_all | default(True) | bool
```

Comment:
I'm good with multiple blocks, and even the conditions being applied to individual tasks where needed for ordering.